src/documenteer/sphinxext/lssttasks/topics.py

```diff
@@ -131,12 +131,19 @@ class TaskTopicDirective(BaseTopicDirective):
     """
 
     def get_type(self, class_name):
-        from lsst.pipe.base import CmdLineTask, PipelineTask
+        try:
+            from lsst.pipe.base import CmdLineTask
+        except ImportError:
+            CmdLineTask = None
```

Comment:
Hmm, maybe if I set this to `type(None)` then I won't need the 'is not None' below since the `issubclass` would work and always return False.

Reply:
I think it's fine as is.

Reply:
I will change it. My `mypy` demon in my head is telling me that `CmdLineTask` must be a `type`.

Reply:
Okay. I won't change it since you've merged it now.

Reply:
I'll fix it if it becomes a problem and the code is still around.